src/test/java/org/primeframework/mvc/BrowserSessionTest.java

```diff
@@ -63,7 +90,7 @@ public void sessionCookie() throws Exception {
                                  .post()
                                  .assertStatusCode(302)
                                  .assertRedirect("/browser-session/second")
-                                 .assertContainsCookie("user")) // Delete the cookie
+                                 .assertCookieWasDeleted("user")) // Delete the cookie
```

Comment:
Interesting. Was contains technically passing since a "deletion" 'sets' the cookie to null, but the test should have been more specifically asserting a null value?

Reply:
Yeah. I think that any action using a managed cookie will include the `Set-Cookie` header on the response even if it doesn't change the value. This seems more correct for this test.